packages/office-ui-fabric-react/src/components/Button/CommandButton/CommandButton.scss

```diff
@@ -9,61 +9,61 @@
 
 $button-commandButtonHeight: 40px;
 
-.ms-Button--command {
-  @include text-align(left);
+.ms-Button {
+  &.ms-Button--command {
```

Comment:
@mikewheaton Yes, without this, there is an amount of non-determinism whether the ms-button-- Command set of styles actually apply. Depending on load order, the ms-button (ButtonCore) changes overwrite the Command button styles, which is clearly not intended...